locales/fr/json.json

```diff
@@ -88,8 +88,8 @@
     "Show All": "Afficher tout",
     "Sign In": "Se connecter",
     "Solve": "Résoudre",
-    "Start": "Start",
-    "Stop": "Stop",
+    "Start": "Commencer",
+    "Stop": "Arrêt",
```

Comment:
Arrêter

Reply:
Merci

New PR is https://github.com/Laravel-Lang/actions/pull/1115